internal/types/expression.go

```diff
@@ -68,15 +68,23 @@ func (e *FieldPathError) Code() ExpressionErrorCode {
 // Expression is an expression constructed from field value.
 type Expression interface {
 	Evaluate(doc *Document) any
+	GetPath() Path
 }
```

Comment:
I don't quiet understand why do we have interface for that. Is there any other expression that could implement that interface and be used?

Reply:
Current usage is https://github.com/FerretDB/FerretDB/pull/2294/files#diff-1718afa874e27b028718639f7add6114f09306e118cd3080bad050b4c806b8b2R114

I suggest renaming to `GetExpressionSuffix()` or something else and return string. So we don't have to expose detail about `Path`. What do you think?

Reply:
Thanks for explanation!

> I suggest renaming to GetExpressionSuffix() or something else and return string.

It looks to me that other expressions might not implement suffix. What would be the suffix of system variable for example?

Reply:
https://www.mongodb.com/docs/manual/meta/aggregation-quick-reference/#aggregation-variables

Maybe just calling `GetExpressionValue()` and returning `all.path.value` for path might be better. That way, for variable expression, it can return `NOW` for `$$NOW` variable.

The caveat is that it has to then find suffix in the callers code.

Reply:
That's a good idea, thanks!

Reply:
Done